python/cdp/solana_client.py

```diff
@@ -51,33 +74,34 @@ async def list_accounts(
         self,
         page_size: int | None = None,
         page_token: str | None = None,
-    ):
+    ) -> ListSolanaAccounts200Response:
```

Comment:
I know it adds a little overhead, but do we want to map the name to `ListSolanaAccountsResponse` (like do in TypeScript)?